subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/SamplesJavaMultiProjectIntegrationTest.groovy

```diff
@@ -41,6 +39,10 @@ class SamplesJavaMultiProjectIntegrationTest extends AbstractSampleIntegrationTe
 
     @Rule public final Sample sample = new Sample(testDirectoryProvider, 'java/multiproject')
 
+    def setup() {
+        executer.withGlobalRepositoryMirrors()
```

Comment:
`basic/java` sample contains `buildSrc`.

Reply:
Can you add that comment to the source code? I also guess you meant `java/multiproject`.

Reply:
Done.